examples/configs/abc_streaming/config.yml

```diff
@@ -0,0 +1,40 @@
+instructions:
+  - type: general
+    content: |
+      Below is a conversation between a user and a bot called the ABC Bot.
+      The bot is designed to answer employee questions about the ABC Company.
+      The bot is knowledgeable about the employee handbook and company policies.
+      If the bot does not know the answer to a question, it truthfully says it does not know.
+
+sample_conversation: |
+  user "Hi there. Can you help me with some questions I have about the company?"
+    express greeting and ask for assistance
+  bot express greeting and confirm and offer assistance
+    "Hi there! I'm here to help answer any questions you may have about the ABC Company. What would you like to know?"
+  user "What's the company policy on paid time off?"
+    ask question about benefits
+  bot respond to question about benefits
+    "The ABC Company provides eligible employees with up to two weeks of paid vacation time per year, as well as five paid sick days per year. Please refer to the employee handbook for more information."
+
+models:
+  - type: main
+    engine: openai
+    model: gpt-3.5-turbo-instruct
+
+rails:
+  input:
+    flows:
+      - self check input
+
+  output:
+    flows:
+      - self check output
+    streaming:
+      enabled: True
+      window_size: 20
+
+  dialog:
+    single_call:
+      enabled: False
+
+streaming: True
```

Comment:
Any chance this top-level `streaming` field seems problematic in light of the new `rails.output.streaming.enabled` field?  (Reading...maybe I'll figure it out.)

Reply:
Yes! If you set it to false then streaming is disabled (no matter what we have for output rails streaming). 

Right now, we can enable/disable streaming for all output rails, but in future we might want to make it configurable per output rails so having something global makes sense. What do you think?

Reply:
My pref is for a single control and the per-output rail config seemed OK to me.  It's not a deal-breaker to have two knobs to twist, but it doesn't seem like a feature either.

Reply:
I see your point, let's see if I understand it


You think, this is easier? I think it is

```yaml
  output:
    flows:
      - self check output
    streaming:
      window_size: 20

  dialog:
    single_call:
      enabled: False

streaming: True
```

and we can have defaults so that following also works:

```yaml
  output:
    flows:
      - self check output

  dialog:
    single_call:
      enabled: False

streaming: True

```

I can change it to work like this.

@cparisien would this be OK?

Reply:
I'm ok with this.